foodalert/static/foodalert/js/components/audit-controller.vue

```diff
@@ -1,12 +1,143 @@
 <template>
-    <audit-template></audit-template>
+    <audit-template
+        :items="this.filteredItems"
+        :years="this.years"
+        :months="this.months"
+        @export="this.exportTable"
+        @updateMonth="this.updateMonth"
+        @updateYear="this.updateYear"
+        >
+    </audit-template>
 </template>
 
 <script type="text/javascript">
     import AuditTemplate from './audit-template.vue';
+    const axios = require('axios')
```

Comment:
We should use synchronous imports instead of async. Both for the sake of consistency and webpack optimization